atomicapp/nulecule/container.py

```diff
@@ -182,6 +182,10 @@ def extract_nulecule_data(self, image, source, dest, update=False):
         logger.debug('Removing tmp dir: %s' % tmpdir)
         Utils.rm_dir(tmpdir)
 
+        # Set the proper permissions on the extracted folder
+        logger.debug("Setting file ownership on %s" % dest)
```

Comment:
would it be better to have this debug statement in the setFileOwnerGroup function?  Then we could have it say "Setting owner/group of %s to %s/%s"